consensus/ibft/src/main/java/tech/pegasys/pantheon/consensus/ibft/validation/NewRoundMessageValidator.java

```diff
@@ -121,9 +123,11 @@ private boolean validateRoundChangeMessagesAndEnsureTargetRoundMatchesRoot(
 
     for (final SignedData<RoundChangePayload> roundChangeMsg :
         roundChangeCert.getRoundChangePayloads()) {
+      // RoundChangeValidator requires the "requisite Prepare Msg Count" (which is 1 less than
```

Comment:
Nit: this comment could be incorporated into a method or constant, negating the need for its existence.

Reply:
Has been moved out to IbftHelpers.prepareMessageCountFromQuorum()